pkg/dogstatsd/server_test.go

```diff
@@ -554,8 +559,9 @@ func TestE2EParsing(t *testing.T) {
 	config.Datadog.SetDefault("dogstatsd_port", port)
 
 	demux := aggregator.InitTestAgentDemultiplexerWithFlushInterval(10 * time.Millisecond)
-	s, err := NewServer(demux, false)
+	s := NewServer(false)
 	require.NoError(t, err, "cannot start DSD")
+	_ = s.Start(demux)
```

Comment:
Would it make sense to check for error here as well? (this pattern repeats a few times)

Maybe a helper function, something like `func MustStart(t *testing.T, bool) Component`, would make this simpler?

Reply:
Arg I missed some. Yeah - a helper is a good idea - ill do that.